test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -1103,8 +1103,8 @@ protected static void wipeAllIndices() throws IOException {
     protected static void wipeAllIndices(boolean preserveSecurityIndices) throws IOException {
         boolean includeHidden = clusterHasFeature(RestTestLegacyFeatures.HIDDEN_INDICES_SUPPORTED);
         try {
-            // remove all indices except ilm and slm history which can pop up after deleting all data streams but shouldn't interfere
-            final List<String> indexPatterns = new ArrayList<>(List.of("*", "-.ds-ilm-history-*", "-.ds-.slm-history-*"));
+            // remove all indices except data stream backing indices, since they have to be removed their own special way, anyway
```

Comment:
Should we just add `-.ds-.watcher-history-*` as a smaller step that would still fix these two tests? The problem is that watcher history is written asynchronously. That's not necessarily the case with _every_ datastream

Reply:
Sure, that's fine. I'll change it to that.

Reply:
Done via d353c5b72d604abf726b62f056eceed1ee0e7031.